<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Madhus-MacBook-Air.local" failures="1" tests="4" name="Sales Force Test Case" time="13.638" errors="0" timestamp="2024-12-13T20:17:03 GMT-06:00">
  <testcase classname="com.salesforce.testcases.SFDCLoginTestCases" name="PasswordFeildEmptyTest" time="1.59"/>
  <testcase classname="com.salesforce.testcases.SFDCLoginTestCases" name="ValidLoginTest" time="2.854">
    <failure type="java.lang.AssertionError" message="TestCase02 passed expected [Home Page ~ Salesforce - Developer Edition] but found []">
      <![CDATA[java.lang.AssertionError: TestCase02 passed expected [Home Page ~ Salesforce - Developer Edition] but found []
at com.salesforce.testcases.SFDCLoginTestCases.ValidLoginTest(SFDCLoginTestCases.java:60)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- ValidLoginTest -->
  <testcase classname="com.salesforce.testcases.SFDCLoginTestCases" name="RemberMeChkBoxTest" time="2.655"/>
  <testcase classname="com.salesforce.testcases.SFDCLoginTestCases" name="CheckforIncorrectCrdentials" time="0.378"/>
</testsuite> <!-- Sales Force Test Case -->
